### DISCUSSION TITLE
Employment certificate in local language only

### DISCUSSION THREAD

**Discussion by devedzic** [community/community#138144] [GitHub Education]
### Hi everyone,

Hi,

I am a teacher at the University of Belgrade, Serbia. My employment contract is in local language (Serbian) only. Can I use it for applying for GitHub Education as a teacher? My university does not issue any teacher cards. 

I have attached my employment contract in its current form for your reference.

Note that I have already tried to apply with an employment certificate edited in English and printed on my school's letterhead, but it was not accepted as it was edited.

Thank you.

[Vladan Devedzic - Ugovor o radu, 26.06.2015.pdf](https://github.com/user-attachments/files/16923912/Vladan.Devedzic.-.Ugovor.o.radu.26.06.2015.pdf)

**Comment by VadymMakohon** [comment]
When applying for GitHub Education as a teacher, GitHub typically requires documentation in English or an official translation if the document is in another language, like your Serbian employment contract. Since your previous application with an edited employment certificate was not accepted, I would recommend the following approach:

Suggestions for Applying:
Use the Original Serbian Contract:

You can submit your employment contract in its original Serbian form. While it is not in English, GitHub support may still review it if you explain your situation. Make sure it’s the official document issued by your university.
Provide an Official Translation:

If possible, get an official translation of your employment contract. This should be a certified translation by an official translator, which GitHub will likely accept since it hasn’t been "edited" by you. You can submit both the Serbian original and the certified translation.
Attach Additional Supporting Documents:

If your university doesn't issue teacher ID cards, mention that in your application and include other official documentation such as:
A link to your university profile page showing your name and title.
A letter from your department confirming your employment, signed by a university official, even if it's in Serbian.
Pay stubs or official correspondence from the university that shows your employment status.
Explain the Situation in the Application:

Clearly explain that your university does not issue teacher cards and that your employment contract is in Serbian. You can state that you’ve provided the official document in the local language and ask for consideration based on that.
This combination of your Serbian contract and a certified translation (if possible) should improve your chances of getting approved for GitHub Education as a teacher.

**Reply by devedzic** [reply]
Dear Vadym,
Thanks for your prompt answer :)
Fine, I can send a snapshot of my employment contract in Serbian, BUT:
> GitHub support may still review it if you explain your situation
WHERE EXACTLY to explain it? In the application process, I only see the options attached as a screenshot to this note. There is no "comment box" or something where I can explain it in words.
Thank you in advance on advising me on this.
Vladan

![Capture](https://github.com/user-attachments/assets/e94686b3-6ee1-4969-afc2-f3b3bef9bd74)

**Reply by VadymMakohon** [reply]
Hmmm.
Try to do either:
1. Follow-Up via Existing Ticket:
Reply to Your Existing Ticket: If you received an email confirmation or ticket number when you submitted your initial request, reply to that email with additional details about your situation. Most support systems allow replies to existing tickets.
Submit a New Request with Detailed Information:
Use the GitHub Support Contact Form: If the current ticket doesn’t allow additional explanations, you might need to submit a new support request.
2. Go to the GitHub Support Contact Page.
Select "Account Recovery" or the appropriate category.
When filling out the form, provide as much detailed information as possible about your situation, even if there isn’t a dedicated comment box. You might need to include detailed explanations in the provided fields.

**Comment by devedzic** [comment]
Dear Vadym,
Thanks again for your prompt reply.
Please note that I already tried replying to the email I received from GitHub, but it was bounced (which I expected - the address is edu-**noreply**@github.com).
Also, there is no "Account Recovery" category on the GitHub Support Contact page. I have created another ticket in another category. I am waiting for their reply.
Thanks for all of your help so far.
Vladan
